snuba/datasets/configuration/issues/entities/search_issues.yaml

```diff
@@ -91,6 +91,63 @@ translation_mappers:
         from_col_name: "timestamp"
         to_table_name:
         to_col_name: "client_timestamp"
+    - mapper: ColumnToColumn
+      args:
+        from_table_name:
+        from_col_name: "message"
+        to_table_name:
+        to_col_name: "search_title"
+    - mapper: ColumnToColumn
+      args:
+        from_table_name:
+        from_col_name: "title"
+        to_table_name:
+        to_col_name: "search_title"
+    - mapper: ColumnToIPAddress
+      args:
+        from_table_name:
+        from_col_name: "ip_address"
+        to_function_name: "coalesce"
```

Comment:
This needs to be coalescing the ipv4 and ipv6 cols right? Not sure if that's possible in this yaml format

Reply:
Nm, just realised `ColumnToIPAddress` is a magic translator

Reply:
I think its baked into the ColumnToIPAddress mapper:
https://github.com/getsentry/snuba/blob/306097c69963868c9381f806f77afee9104a7b3c/snuba/clickhouse/translators/snuba/mappers.py#L120-L134